### PR TITLE
add created fields to tag collections

### PR DESCRIPTION
**Pull request by darkruby501**

### CODE REVIEW

File: packages/lesswrong/lib/collections/tagRels/collection.ts [hunk 1]
```diff
@@ -5,6 +5,12 @@ import { makeVoteable } from '../../make_voteable';
 import { userCanUseTags } from '../../betas';
 
 const schema = {
+  createdAt: {
+    optional: true,
+    type: Date,
+    canRead: ['admins'],
```
Comment (Discordius) [inline packages/lesswrong/lib/collections/tagRels/collection.ts:11]: I think they should just be readable by guests, all the other createdAt fields are readably by guests

File: packages/lesswrong/lib/collections/tags/collection.ts [hunk 2]
```diff
@@ -15,6 +15,12 @@ const formGroups = {
 };
 
 const schema = {
+  createdAt: {
+    optional: true,
+    type: Date,
+    canRead: ['admins'],
```
Comment (Discordius) [inline packages/lesswrong/lib/collections/tags/collection.ts:21]: Same as the other thing

Comment (Raemon) [reply packages/lesswrong/lib/collections/tags/collection.ts:21]: Seems fine, but bear in mind these aren't in any fragments and don't show up anywhere

Comment (Discordius) [reply packages/lesswrong/lib/collections/tags/collection.ts:21]: Yep, seems fine. I expect we will add this field to some fragments later.

Comment (darkruby501) [reply packages/lesswrong/lib/collections/tags/collection.ts]: Requested changes have been made.

Comment (Raemon) [reply packages/lesswrong/lib/collections/tags/collection.ts]: (note: the current procedure here is to set the PR's "assigned" status to whoever asked for the changes.

Comment (Raemon) [reply packages/lesswrong/lib/collections/tags/collection.ts]: (if someone requests changes, that person then has to be the one to approve the PR)

Comment (darkruby501) [reply packages/lesswrong/lib/collections/tags/collection.ts]: +1

Comment (darkruby501) [reply packages/lesswrong/lib/collections/tags/collection.ts]: @Discordius

**Pull request merged**